gattc_windows.go

```diff
@@ -360,14 +361,29 @@ func (c DeviceCharacteristic) Read(data []byte) (int, error) {
 	return len(readBuffer), nil
 }
 
+// EnableNotifications enables notifications in the Client Characteristic
+// Configuration Descriptor (CCCD). Just the Notify mode is supported.
+func (c DeviceCharacteristic) EnableNotifications(callback func(buf []byte)) error {
```

Comment:
`EnableNotifications` enables notifications or indications, whatever is available. And it favors notifications over indications. This PR should not change that behaviour.

Reply:
Sorry， i will update to keep the original behavior